docker-compose.yml

```diff
@@ -2,29 +2,25 @@ version: "3.5"
 
 networks:
   app-network:
-    name: app-network
```

Comment:
because docker-compose will name everything `projectname-resource`, where resource is in this case `app-network`.

```
networks:
  app-network:
    ...
```

So you will have: `inception-app-network`